pkg/proxy/proxyports/proxyports.go

```diff
@@ -119,7 +118,7 @@ type ProxyPorts struct {
 func NewProxyPorts(
 	logger *slog.Logger,
 	config ProxyPortsConfig,
-	datapathUpdater datapath.IptablesManager,
+	datapathUpdater DatapathUpdater,
```

Comment:
```
time=2025-07-22T10:12:27.143263331Z level=error source=/go/src/github.com/cilium/cilium/vendor/github.com/cilium/hive/cell/invoke.go:52 msg="Invoke failed" error="missing type:\n\t- proxyports.DatapathUpdater (did you mean to Provide it?)" function="cmd.configureAPIServer (cmd/cells.go:347)"
```

`DatapathUpdater` is not provided

Reply:
`ENOTENOUGHSLEEP` :facepalm: that's probably not worth the churn then. I'll drop the commit.